### PR TITLE
fix(firestore): disable firestore cache globally

### PR DESCRIPTION
**Pull request by yoannLafore**
This PR aims to provide the temporary fix to #78 of disabling firestore cache.

Concretely, the firestore cache is globally disabled in the `main` function of the application.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -11,6 +12,15 @@ void main() async {
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
   );
+
+  // TODO: Remove this line when the issue is correctly fixed (https://github.com/ProximaEPFL/proxima/issues/78)
```
Comment (JoachimFavre) [inline lib/main.dart:16]: Issue #78 will close when this PR (#79) is merged. I don't think there is something necessary to change to reflect this, but I just wanted to point it out.

**Review by JoachimFavre — COMMENTED** [review]
LGTM! I just added a small observation that I think is important.

It's really nice removing the cache fixed the bug. Good job :)

File: lib/main.dart (hunk 1 above)
Comment (yoannLafore) [reply lib/main.dart:16]: You are right, I didn't though of that.
I think what we can do is to reopen the issue #78 after this PR is merged.

**Review by TheTexanCodeur — APPROVED** [review]
LGTM!

We just need to keep in mind that this only a temporary fix for now. This is why i think that he todo in the code is a good idea  We won't forget about this point in the future thanks to this comment.

**Comment by JoachimFavre** [comment]
> We just need to keep in mind that this only a temporary fix for now. This is why i think that he todo in the code is a good idea We won't forget about this point in the future thanks to this comment.

I think we should also put this as an issue in the backlog. This is vital for the code to be usable offline.

**Comment by yoannLafore** [comment]
I agree with you, issue #78 has been put into the product backlog.

**Review by JoachimFavre — APPROVED** [review]
Everything looks perfect now, just make sure to keep the issue open :)

**Pull request merged**